src/core/trtserver.cc

```diff
@@ -484,6 +486,22 @@ TrtServerRequestOptions::SetBatchSize(uint64_t batch_size)
   return nullptr;  // Success
 }
 
+TRTSERVER_Error*
+TrtServerRequestOptions::SetPriority(uint32_t priority)
+{
+  std::lock_guard<std::mutex> lk(mtx_);
+  request_header_->set_priority(priority);
+  return nullptr;  // Success
+}
+
+TRTSERVER_Error*
+TrtServerRequestOptions::SetTimeoutMs(uint64_t timeout_ms)
+{
+  std::lock_guard<std::mutex> lk(mtx_);
+  request_header_->set_timeout_microseconds(timeout_ms);
```

Comment:
With `ms` I'd assume milliseconds.  Consider using `us` if it's really microseconds.